### PR TITLE
bitcoind_: update for Bitcoin Core v0.19.0 release

### LINKED ISSUE #995: bitcoind_: LookupError: 'base64' is not a text encoding; use codecs.encode() to handle arbitrary codecs
I've just tried updated `plugins/currency/bitcoin/bitcoind_`, and I get python exception:
```
env rpcpassword=??? rpcuser=??? bitcoin_configfile=/etc/bitcoin/bitcoin.conf ./bitcoind_ autoconf
Traceback (most recent call last):
  File "./bitcoind_", line 267, in <module>
    main()
  File "./bitcoind_", line 120, in main
    (info, error) = bitcoin.getinfo()
  File "./bitcoind_", line 240, in __call__
    b64 = ('%s:%s' % (self.service.username, self.service.password)).encode('base64')[:-1]
LookupError: 'base64' is not a text encoding; use codecs.encode() to handle arbitrary codecs
```

My setup:
```
# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.9 (stretch)
Release:        9.9
Codename:       stretch
```
```
# python3 --version
Python 3.5.3
```

@sumpfralle maybe you tested it with more recent Python version?

### PR DESCRIPTION
**Pull request by Talkless**
* Use new RPC calls to get relevant statistics, as some calls are either
deprecated or completely removed.

* Fix response printing when MUNIN_DEBUG is set.

* Remove "fees" graph as there is no sane way to implement that.

Closes #995

### CODE REVIEW

**Comment by Talkless** [comment]
@sumpfralle Could you help me with these tests? I don't see any test code in plugin, what's expected from the tests?

**Comment by sumpfralle** [comment]
It is slightly hidden:
```
plugins/currency/bitcoin/bitcoind_:168:1: E302 expected 2 blank lines, found 1
```

`flake8` is picky about empty lines (just as [PEP8](https://www.python.org/dev/peps/pep-0008/)) :)

File: plugins/currency/bitcoin/bitcoind_ [hunk 1]
```diff
@@ -134,20 +134,36 @@ def main():
         print(error, file=sys.stderr)
         return False
 
+    if request_var == 'balance':
+        # we use getbalance*s* (plural) as old betbalance is being deprecated,
```
Comment (sumpfralle) [inline plugins/currency/bitcoin/bitcoind_:138]: *old getbalance*, I guess?

File: plugins/currency/bitcoin/bitcoind_ [hunk 2]
```diff
@@ -134,20 +134,36 @@ def main():
...
+        # and we have to calculate total balance (owned and watch-only) manually now.
+        (info, error) = bitcoin.getbalances()
+
+        total = 0
+
+        for wallet_mode in ('mine', 'watchonly'):
+            if wallet_mode in info:
+                total += info[wallet_mode]['trusted']
+
```
Comment (sumpfralle) [inline plugins/currency/bitcoin/bitcoind_:147]: The five lines above could be beautifully combined into a list comprehension:
```python
total = sum(info[wallet_mode]['trusted']
            for wallet_mode in ('mine', 'watchonly')
            if wallet_mode in info)
```

Sorry in advance for lecturing - I just like python's comprehensions a lot ...

(of course, it is just a matter of taste)

Comment (Talkless) [reply plugins/currency/bitcoin/bitcoind_:145]: I like using algorithms too in C++ ;) . Fixed.

**Comment by Talkless** [comment]
> If not: could it be simply detect the version and react appropriately?

Ugh.. well.. It could I guess. 

First we could query that new `getnetworkinfo`, and if it returns `404`, retry with deprecated `getinfo`. 

Maybe it would be useful to parse minor version (like 19, 18, upcomming 20, etc) so it would be easy to fix issues with new versions alter.

**Comment by Talkless** [comment]
@sumpfralle I've returned old version support. Tested with v0.15 and v0.19.

**Comment by sumpfralle** [comment]
> I've returned old version support. Tested with v0.15 and v0.19.

Great!

Three minor style issue (to be checked via `flake8`) are left:
```
# plugins/currency/bitcoin/bitcoind_:65:1: E302 expected 2 blank lines, found 1
# plugins/currency/bitcoin/bitcoind_:79:47: E261 at least two spaces before inline comment
# plugins/currency/bitcoin/bitcoind_:85:100: E501 line too long (143 > 99 characters)
```

Thank you!

**Comment by Talkless** [comment]
@sumpfralle Ping, I've pushed flake8 issues.

**Comment by sumpfralle** [comment]
Thank you!

**Pull request merged**